src/components/ClientForm/index.jsx

```diff
@@ -143,6 +143,17 @@ export default class ClientForm extends Component {
               />
             </FormGroup>
           </ListItem>
+          {isNewClient ? (
```

Comment:
`{isNewClient && (`

Reply:
Wouldn't this actually display "False" to the view when it's not a new client?

Reply:
OK, maybe not :)